# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/mathml/nsMathMLChar.cpp
# Commit: 62f8ce89dce8
# Full Hash: 62f8ce89dce8f1162ff2f57debdd9a756a11a2f9
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-04-03 01:02:14
# Regressor Bug: 1619349
# File Overlap Count: 1
# Description:
#   Bug 1619349 - patch 1 - Record statistics about font-matching behavior in the presContext. r=jwatt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D67787
# ==============================================================================

diff -r bcbee7191130 -r 62f8ce89dce8 layout/mathml/nsMathMLChar.cpp
--- a/layout/mathml/nsMathMLChar.cpp	Thu Apr 02 01:38:21 2020 +0000
+++ b/layout/mathml/nsMathMLChar.cpp	Thu Apr 02 01:38:18 2020 +0000
@@ -885,6 +885,7 @@
     params.explicitLanguage = styleFont->mExplicitLanguage;
     params.userFontSet = aPresContext->GetUserFontSet();
     params.textPerf = aPresContext->GetTextPerfMetrics();
+    params.fontStats = aPresContext->GetFontMatchingStats();
     params.featureValueLookup = aPresContext->GetFontFeatureValuesLookup();
     RefPtr<nsFontMetrics> fm =
         aPresContext->DeviceContext()->GetMetricsFor(font, params);
@@ -1398,6 +1399,7 @@
   params.explicitLanguage = styleFont->mExplicitLanguage;
   params.userFontSet = presContext->GetUserFontSet();
   params.textPerf = presContext->GetTextPerfMetrics();
+  params.fontStats = presContext->GetFontMatchingStats();
   RefPtr<nsFontMetrics> fm =
       presContext->DeviceContext()->GetMetricsFor(font, params);
   uint32_t len = uint32_t(mData.Length());
